src/blocks/ImageTextBlockBrandPivot/index.tsx

```diff
@@ -178,11 +179,31 @@ const ImageVideo = styled(DeferredVideo)({
   borderRadius: 0.01,
 })
 
+const Wordmark = styled('div')({
+  display: 'inline-flex',
+  position: 'absolute',
+  marginTop: '0.2rem',
+  marginLeft: '0.5rem',
+
+  ['svg']: {
+    width: '2rem',
+    height: '2rem',
+  },
+
+  [TABLET_BP_DOWN]: {
+    ['svg']: {
+      width: '1.25rem',
+      height: '1.25rem',
+    },
+  },
```

Comment:
When refactoring I've seen we have a mix of max-width and min-width media queries. I think it's nice if we use a mobile first approach and use min-width media queries :)
```suggestion
  ['svg']: {
    width: '1.25rem',
    height: '1.25rem',
  },

  [TABLET_BP_UP]: {
    ['svg']: {
      width: '2rem',
      height: '2rem',
    },
  },
```